aspnetcore/signalr/publish-to-azure-web-app.md

```diff
@@ -1,63 +1,80 @@
 ---
-title: Publish an ASP.NET Core SignalR app to Azure Web App
+title: Publish an ASP.NET Core SignalR app to Azure App Service
 author: bradygaster
-description: Publish an ASP.NET Core SignalR app to Azure Web App
+description: Learn how to publish an ASP.NET Core SignalR app to the Azure App Service.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: bradyg
 ms.custom: mvc
-ms.date: 04/20/2018
+ms.date: 06/25/2019
 uid: signalr/publish-to-azure-web-app
 ---
+# Publish an ASP.NET Core SignalR app to Azure App Service
 
-# Publish an ASP.NET Core SignalR app to an Azure Web App
+By [Brady Gaster](https://twitter.com/bradygaster)
 
-[Azure Web App](/azure/app-service/app-service-web-overview) is a [Microsoft cloud computing](https://azure.microsoft.com/) platform service for hosting web apps, including ASP.NET Core.
+[Azure App Service](/azure/app-service/app-service-web-overview) is a [Microsoft cloud computing](https://azure.microsoft.com/) platform service for hosting web apps, including ASP.NET Core.
 
 > [!NOTE]
-> This article refers to publishing an ASP.NET Core SignalR app from Visual Studio. Visit [SignalR service for Azure](https://azure.microsoft.com/services/signalr-service) for more information about using SignalR on Azure.
+> This article refers to publishing an ASP.NET Core SignalR app from Visual Studio. For more information, see [SignalR service for Azure](https://azure.microsoft.com/services/signalr-service).
 
 ## Publish the app
 
-Visual Studio provides built-in tools for publishing to an Azure Web App. Visual Studio Code user can use [Azure CLI](/cli/azure) commands to publish apps to Azure. This article covers publishing using the tools in Visual Studio. To publish an app using Azure CLI, see [Publish an ASP.NET Core app to Azure with command line tools](/azure/app-service/app-service-web-get-started-dotnet).
+This article covers publishing using the tools in Visual Studio. Visual Studio Code users can use [Azure CLI](/cli/azure) commands to publish apps to Azure. For more information, see [Publish an ASP.NET Core app to Azure with command line tools](/azure/app-service/app-service-web-get-started-dotnet).
 
-Right-click on the project in **Solution Explorer** and select **Publish**. Confirm that **Create new** is checked in the **Pick a publish target** dialog, and select **Publish**.
+1. Right-click on the project in **Solution Explorer** and select **Publish**. Confirm that **App Service** and **Create new** are selected in the **Pick a publish target** dialog. Select **Create Profile** from the **Publish** button drop down.
 
-![Pick publish target](publish-to-azure-web-app/_static/pick-publish-target-dialog.png)
+   Enter the information described in the following table in the **Create App Service** dialog and select **Create**.
```

Comment:
Later on the doc instructions assume that this doc is general enough to cover deploying to an existing App Service instance, not just a new one.

Reply:
That's the way that Brady wrote it. He would need to address this because idk what the process is if the App Service pre-exists and/or the Publish Profile pre-exists without setting the added dependency in the UI.

Reply:
For consistency, then, I would leave this part and just change the later text so it doesn't say new or existing.